internal/controller/driver_controller.go

```diff
@@ -1049,6 +1195,9 @@ func (r *driverReconcile) reconcileNodePluginDeamonSet() error {
 							utils.PodsMountDirVolume(kubeletDirPath),
 							utils.RegistrationDirVolume(kubeletDirPath),
 						)
+						if logRotator != nil {
+							volumes = append(volumes, utils.LogsDirVolume(logHostPath, daemonSet.Name), utils.LogRotateDirVolumeName(r.driver.Name))
+						}
```

Comment:
Can you please move this to be the last volume added?
And make it a multi-line statement (the line is very long)